src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -14127,6 +14148,10 @@ void func_80852608(PlayState* play, Player* this, CsCmdActorAction* arg2) {
     func_808322D0(play, this, &gPlayerAnim_002838);
 }
 
+/**
+ * Unequip Master Sword and remove from inventory. Since this only checks on frame 10 of the animation, if the animation
+ * never reaches this frame (such as if stuck in a "superslide" state), the sword will not be removed.
+ */
```

Comment:
not sure about mentioning the "bug" of the superslide state in the function comment. seems more appropriate at the check for the frame